tools/InstallModule.ps1

```diff
@@ -0,0 +1,47 @@
+[cmdletbinding()]
+param(
+  [string]
+  [Parameter(Mandatory = $false, Position = 0)]
+  $ModuleName = "Az",
+  [string]
+  [Parameter(Mandatory = $false, Position = 1)]
+  $SourceLocation = $PSScriptRoot
+)
+
+function Register-Gallery {
+  param(
+    [string]
+    [Parameter(Mandatory = $true, Position = 0)]
+    $GalleryName,
+    [string]
+    [Parameter(Mandatory = $false, Position = 1)]
+    $SourceLocation
+  )  
+  switch ($GalleryName) {
+    'PSGallery' {   
+      Write-Host "Setting $GalleryName Trusted..."
+      Set-PSRepository -Name $GalleryName -InstallationPolicy Trusted
+      break;
+    }
+    'TestGallery' { 
+      Write-Host "Registering $GalleryName Trusted..."
+      Register-PSRepository -Name $GalleryName -SourceLocation 'https://www.poshtestgallery.com/api/v2' -PackageManagementProvider NuGet -InstallationPolicy Trusted
+      break;
+    }
+    Default {
+      Write-Host "Registering $GalleryName Trusted..."
```

Comment:
As this will be displayed to the user, I suggest we make it more clear
```suggestion
      Write-Host "Registering temporary repository $GalleryName with InstallationPolicy Trusted..."
```